projects/plugins/jetpack/_inc/lib/admin-pages/class.jetpack-react-page.php

```diff
@@ -181,6 +181,37 @@ function page_admin_scripts() {
 		wp_add_inline_script( 'react-plugin', 'var jetpack_redirects = { currentSiteRawUrl: "' . $site_suffix . '" };', 'before' );
 	}
 
+	/**
+	 * Gets a purchase token that is used for Jetpack logged out visitor checkout.
+	 * The purchase token should be appended to all CTA url's that lead to checkout.
+	 *
+	 * @since 9.6.3
+	 * @return string|boolean
+	 */
+	public function getPurchaseToken() {
+		$purchase_token = Jetpack_Options::get_option( 'purchase_token', false );
```

Comment:
We need to bail early in this method when the current user is not the main/master/connecting user.